README.md

```
@@ -56,4 +56,8 @@ To test the extension locally:
 ###🔚 To deactivate the virtual environment
 ```bash
 deactivate
-```
\ No newline at end of file
+```
+
+#### 📝 useful links for reference:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)